src/web/app/components/teammates-common/submission-status-name.pipe.ts

```diff
@@ -12,20 +12,49 @@ export class SubmissionStatusNamePipe implements PipeTransform {
   /**
    * Transforms {@link FeedbackSessionSubmissionStatus} to a simple name.
    */
-  transform(status: FeedbackSessionSubmissionStatus): string {
+  transform(status: FeedbackSessionSubmissionStatus, deadlines?: {
+    studentDeadlines: Record<string, number>, instructorDeadlines: Record<string, number>
+  }): string {
+    let string = '';
     switch (status) {
       case FeedbackSessionSubmissionStatus.NOT_VISIBLE:
       case FeedbackSessionSubmissionStatus.VISIBLE_NOT_OPEN:
-        return 'Awaiting';
+        string += 'Awaiting';
+        break;
       case FeedbackSessionSubmissionStatus.OPEN:
-        return 'Open';
+        string += 'Open';
+        break;
       case FeedbackSessionSubmissionStatus.GRACE_PERIOD:
-        return 'Open (grace period)';
+        string += 'Open (grace period)';
+        break;
       case FeedbackSessionSubmissionStatus.CLOSED:
-        return 'Closed';
+        string += 'Closed';
+        break;
       default:
         return 'Unknown';
     }
-  }
 
+    if (deadlines && this.hasOngoingExtension(deadlines)) {
```

Comment:
Will Open be followed by Ex Ongoing?

Reply:
Yup, I think it's ok to have "Open (Ext. ongoing)", as instructors may still want to see if there's extensions

Reply:
Sounds good